### ISSUE TITLE
[Feature request]:  Hide all other app windows except the desktop background and the icons on the desktop.

### ISSUE THREAD

**Issue by parvez-ahammed** [hikarin522/GlassIt-VSC#47]
[Feature idea]: when the extension is active on vsc it automatically hides all other app windows except the background and hides the icons on the desktop also. It would create a better visibility

**Comment by anthonysanlucas** [comment]
Yeah, it is quite annoying I have to minimize everything when vscode is opened. If it would do it automatically it would be great.

**Comment by parvez-ahammed** [comment]
Yep, I hope the dev gets some time to solve it. It would be a stunning theme to use

**Comment by anthonysanlucas** [comment]
I was checking it but the problem is that vsCode does not have that type of accesses, it is possible to manipulate the windows of the desktop by means of some script but I believe that it is not very viable...
It seems that we will have to keep doing it manually 😔

**Comment by RafaelAngelRamirez** [comment]
@piru72 You can achieve this by shaking the vscode window. There's no need to program it (it's a native function of Windows).

**Comment by darianmorat** [comment]
I think the best solution to this is getting a good windows manager like:
- https://github.com/LGUG2Z/komorebi
https://github.com/FancyWM/fancywm?tab=readme-ov-file
